lib/cc/analyzer/engines_config_builder.rb

```diff
@@ -3,6 +3,19 @@
 module CC
   module Analyzer
     class EnginesConfigBuilder
+      class RegistryAdapter < SimpleDelegator
```

Comment:
This logic could've been done as just a private method here, but I thought the adapter class was nice because it hints that, design-wise, this is the interface `Registry` _should_ have, but I've elected not to go down that route because the current CLI/Builder split of responsibilities makes that difficult and investing the time into addressing the difficulty doesn't make sense when we hope to move to something entirely different (the sub-command approach) eventually.